lib/evss/disability_compensation_form/form4142_processor.rb

```diff
@@ -46,17 +46,17 @@ def initialize(submission, jid)
       # @return [Pathname] the stamped PDF path
       #
       def generate_stamp_pdf
+        submission_date = @submission.created_at.in_time_zone('Central Time (US & Canada)')
```

Comment:
You could also utilize the date assignment in the `received_date` method to avoid setting this value a second time in the `generate_metadata` method.